src-vue/src/plugins/store.ts

```diff
@@ -479,6 +488,42 @@ async function _checkForFlightCoreUpdates(state: FlightCoreStore) {
     }
 }
 
+async function flightcoreUpdateCheck() {
+    const update = await check();
+    if (!update?.available) {
+        console.log("No update available");
+    } else if (update?.available) {
+        console.log("Update available!", update.version, update.body);
+        const accepted = await ask(
+        `Update to ${update.version} is available!\n\nRelease notes: ${update.body}`,
+        {
+            title: "Update Available",
+            kind: "info",
+            okLabel: "Update",
+            cancelLabel: "Cancel",
```

Comment:
Should we translate this?

Reply:
At least the `title` entry, yep; I think we can reuse existing entries for the 3 other items.